docs/src/main/asciidoc/trace.adoc

```diff
@@ -1,4 +1,4 @@
-== Spring Cloud Sleuth
+== Stackdriver Trace
```

Comment:
Maybe add an additional sentence in the first paragraph that mentions stackdriver as it opens with "Spring Cloud Sleuth" while the title is Stackdriver Trace.

Reply:
Done. Take a look at the commit below.